core/src/test/scala/unit/kafka/server/RequestQuotaTest.scala

```diff
@@ -561,80 +558,23 @@ class RequestQuotaTest extends BaseRequestTest {
     }
   }
 
-  private def responseThrottleTime(apiKey: ApiKeys, response: Struct): Int = {
```

Comment:
Glad to see this go. I had done something similar in https://github.com/apache/kafka/pull/7409/files#diff-8259941614685619733d9f458762a0b6L564, but hadn't gotten around to pushing that PR over the line.